cdc/log.cc

```diff
@@ -1180,7 +1180,7 @@ struct process_row_visitor {
                     }
                 };
 
-                std::vector<bytes> buf;
+                std::list<bytes> buf;
```

Comment:
Same comment like with udt_visitor. It seems that buf could live inside map_or_list_visitor and not be exposed outside.